synapse/lib/aha.py

```diff
@@ -738,8 +740,15 @@ def _getAhaUrls(self):
         if ahanetw is None:
             return None
 
+        urls = []
+        for serv in self.dmon.listenservers:
```

Comment:
our normal listenservers are the PF Unix socket + the server which may be configured by `dmon:listen` directive.
We normally store the `dmon:listen` server information in the self.sockaddr value.

Reply:
Should we only include those and ignore any additional ports we may be listening on?

Reply:
Yeah sounds good, setting `aha:urls` if you're adding more listeners seems like the right way to do it. I'll get this updated.